website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,76 @@
+| `/experimental` exports             | <Badge text="3" variant="danger" />  | Any types exported from a sub export with the name `/experimental` is considered for experimental and its API may change at any time. [See experiemental api](#experimental-apis)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Formatter                           | <Badge text="4" variant="note" />    | Formatting shouldn't affect the meaning of the language. Changes to the formatting should be taken with consideration, as changes in formatting may introduce significant and unexpected differences in formatted output, but changes to the formatter output are not considered breaking changes for the purposes of issuing a major version change.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| Bug fixes                           | [See bug](#bugs)                     | Bug fixes will not be considered breaking changes for the purposes of TypeSpec versioning if the existing behavior is _clearly_ broken or unintentional (e.g. if it violates the documented and expected behavior, or if there is no reasonable interpretation of the buggy behavior), even if the change is technically a breaking change. Bug fixes that are technically breaking will be documented as breaking changes in the release notes, but will not introduce a major version revision. However, if a bug's behavior has become so widely used as to be considered a "feature," we will consider the impact of the bugfix and make a reasonable determination to the best of our abilities and depending on the non-breaking mitigation strategies we have at our disposal. We will not break behaviors in stable releases that are known to be widely depended upon, even if we believe they are the result of a bug. |
```

Comment:
I wonder if we should have a specific discussion channel for breaking change bugs

Reply:
yeah, I think we can see depending on how much we see it being a breaking issue and go through some more smooth rollout